docs/components/table/README.md

```diff
@@ -46,37 +47,40 @@
 </template>
 
 <script>
+const items = [
+   {
+     isActive: true,  age: 40, name: { first: 'Dickerson', last: 'Macdonald' }
+   }, {
+     isActive: false, age: 21, name: { first: 'Larsen', last: 'Shaw' }
+   }, {
+     _rowVariant: 'success',
+     isActive: false, age: 9,  name: { first: 'Mitzi', last: 'Navarro' }
+   }, {
+     isActive: false, age: 89, name: { first: 'Geneva', last: 'Wilson' }
+   }, {
+     isActive: true,  age: 38, name: { first: 'Jami', last: 'Carney' }
+   }, {
+     isActive: false, age: 27, name: { first: 'Essie', last: 'Dunlap' }
+   }, {
+     isActive: true,  age: 40, name: { first: 'Dickerson', last: 'Macdonald' }
+   }, {
+     _cellVariants: { age: 'danger', isActive: 'warning' },
+     isActive: true,  age: 87, name: { first: 'Larsen', last: 'Shaw' }
+   }, {
+     isActive: false, age: 26, name: { first: 'Mitzi', last: 'Navarro' }
+   }, {
+     isActive: false, age: 22, name: { first: 'Geneva', last: 'Wilson' }
+   }, {
+     isActive: true,  age: 38, name: { first: 'Jami', last: 'Carney' }
+   }, {
+     isActive: false, age: 27, name: { first: 'Essie', last: 'Dunlap' }
+   }
+];
+
 export default {
   data: {
-    items: [
-      {
-        isActive: true, age: 40, name: { first: 'Dickerson', last: 'Macdonald' }
-      }, {
-        isActive: false, age: 21, name: { first: 'Larsen', last: 'Shaw' }
-      }, {
-        _rowVariant: 'success',
-        isActive: false, age: 9, name: { first: 'Mitzi', last: 'Navarro' }
-      }, {
-        isActive: false, age: 89, name: { first: 'Geneva', last: 'Wilson' }
-      }, {
-        isActive: true, age: 38, name: { first: 'Jami', last: 'Carney' }
-      }, {
-        isActive: false, age: 27, name: { first: 'Essie', last: 'Dunlap' }
-      }, {
-        isActive: true, age: 40, name: { first: 'Dickerson', last: 'Macdonald' }
-      }, {
-        _cellVariants: { age: 'danger', name: 'success' },
-        isActive: false, age: 21, name: { first: 'Larsen', last: 'Shaw' }
-      }, {
-        isActive: false, age: 26, name: { first: 'Mitzi', last: 'Navarro' }
-      }, {
-        isActive: false, age: 22, name: { first: 'Geneva', last: 'Wilson' }
-      }, {
-        isActive: true, age: 38, name: { first: 'Jami', last: 'Carney' }
-      }, {
-        isActive: false, age: 27, name: { first: 'Essie', last: 'Dunlap' }
-      }
-    ],
+    items: items,
+    totalRows: items.length,
```

Comment:
We dont even set initial number = items.length, as returned value from table without active filter will be equal to items.length. IMHO